llvm-external-projects/iree-dialects/lib/Dialect/LinalgExt/Utils/Utils.cpp

```diff
@@ -92,6 +92,46 @@ SmallVector<int64_t> asShapeWithAnyValueAsDynamic(ArrayRef<OpFoldResult> ofrs) {
   return result;
 }
 
+FailureOr<SmallVector<OpFoldResult>>
+getInnerTileSizesOfr(OpBuilder &rewriter, Location loc,
+                     RankedTensorType tensorType,
+                     const MaterializeEncodingInfo &materializeEncodingInfo,
+                     MaterializeEncodingValueFn materializeEncodingValueFn) {
+  ArrayRef<int64_t> staticTileSizes = materializeEncodingInfo.innerTileSizes;
+  if (llvm::all_of(staticTileSizes,
+                   [](int64_t i) { return !ShapedType::isDynamic(i); })) {
+    return SmallVector<OpFoldResult>(
+        llvm::map_range(staticTileSizes, [&](int64_t v) {
+          return rewriter.getI64IntegerAttr(v);
+        }));
```

Comment:
I think this can be something like `getAsOpFoldResult(rewriter.getI64ArrayAttr(staticTileSizes))`.

Reply:
Thanks!